case-lib/lib.sh

```diff
@@ -324,6 +324,10 @@ arecord_opts()
 die()
 {
     dloge "$@"
+
+    # Currently devres debug log is enabled only for kmod test
+    # but this is no harm
+    disable_devres_debug_log
```

Comment:
I don't think this is the right place sorry. Tests can fail without calling `die()`, either because of `set -e` or because or bad design or any other reason. I think this should be in `func_exit_handler()`

Reply:
thank you for double checking, agreed!